specification/databox/resource-manager/readme.nodejs.md

```diff
@@ -7,5 +7,9 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 nodejs:
   azure-arm: true
   package-name: azure-arm-databox
-  output-folder: $(node-sdks-folder)/lib/services/databox/lib
-```
+  package-version: 1.0.0-preview
+  output-folder: $(node-sdks-folder)/lib/services/databox
```

Comment:
Is taking off the trailing /lib the right thing to do here? Thought our generate-metadata settings want to put them in `../`.